bodhi/server/models.py

```diff
@@ -2092,10 +2092,10 @@ def check_karma_thresholds(self, db, agent):
             log.debug('%s locked. Ignoring karma thresholds.' % self.title)
             raise LockedUpdateException
         # Return if the status of the update is not in testing
-        if self.status != UpdateStatus.testing:
+        if self.status not in (UpdateStatus.testing, UpdateStatus.pending):
             return
         # If an update receives negative karma disable autopush
-        if self.autokarma and self._composite_karma[1] != 0:
+        if self.autokarma and self._composite_karma[1] != 0 and self.status is UpdateStatus.testing:
```

Comment:
@bowlofeggs We would want to disable autokarma **only** for testing updates as this is conflicting with https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L2380 for pending updates.
As we would not like to disable autopush for pending updates on unstable karma, rather we want to send it to `obsolete` state https://github.com/fedora-infra/bodhi/blob/develop/bodhi/tests/server/services/test_updates.py#L2397